src/main/kotlin/io/github/mojira/arisa/modules/commands/RemoveContentCommand.kt

```diff
@@ -68,10 +68,12 @@ class RemoveContentCommand(
         val jql = """project != TRASH
             | AND issueFunction IN commented($issueFunctionInner)
             | OR issueFunction IN fileAttached($issueFunctionInner)"""
-            .trimMargin().replace("[\n\r]", "")
+            .trimMargin().replace(Regex("[\n\r]"), "")
+
+        val sanitizedUserName = sanitizeCommentArg(userName)
 
         val ticketIds = when (val either = searchIssues(jql, REMOVABLE_ACTIVITY_CAP)) {
-            is Either.Left -> throw CommandExceptions.CANNOT_QUERY_USER_ACTIVITY.create(sanitizeCommentArg(userName))
+            is Either.Left -> throw CommandExceptions.CANNOT_QUERY_USER_ACTIVITY(sanitizedUserName, jql).create(Unit)
```

Comment:
Could you please change this to call `initCause` on the created exception with the exception store in the `Either`?
Then we will know why the execution is failing.

Reply:
Done 👍